hashes/go

```diff
@@ -1,2 +1,2 @@
-# https://golang.org/dl/go1.18.2.src.tar.gz
-SHA512 (go1.18.2.src.tar.gz) = 9214cbc051cf26b49ab1bd4d8d30b060865944b831578a9ea7fcfe33f84009f77932a39f2948efbfc412b151e4734a3bc1f8332f3bea11b35a912b0e23a4118f
+# https://go.dev/dl/go1.18.6.src.tar.gz
+SHA512 (go1.18.6.src.tar.gz) = 2af66b09bfe033b413eb7603a73a490319bf49fec0a2e20c40350e60b9ef35250a6dc8544c5fc67bd1ede55e242d056e7749f69ef500a38b1efe4b8f93078de3
```

Comment:
Is there a reason we don't want Go 1.19.1?

From reading the 1.19 release notes, there isn't anything breaking and we'd get the benefit of consuming some compiler optimizations

Reply:
> Is there a reason we don't want Go 1.19.1?

No reason, I just needed to draw the scope boundary somewhere and a Go update was bigger than I wanted to tackle. Same story with `GCC`.

I can look at adding it next time along with Rust 1.64.0, which will be out later this week.

Reply:
> I just needed to draw the scope boundary somewhere and a Go update was bigger than I wanted to tackle

Super fair! Good to know! Thanks Ben!!